### PR TITLE
Add a wiki page for storeValue

### PR DESCRIPTION
**Pull request by deepakchethan**
fixes https://github.com/appsmithorg/appsmith/issues/853

### CODE REVIEW

**Comment by deepakchethan** [comment]
Hello @hetunandu, Please check if this is fine

File: function-reference/store-value.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+description: >-
+  This function is used to store key value data within browser local storage. `appsmith.store.key` can be used to fetch the stored data.
+---
+
+# Store Value 
+
+## Signature
+
+```text
+storeValue(key: string, value: any) -> void"
```
Comment (mohanarpit) [inline function-reference/store-value.md:11]: Can you please remove double quote at the end? Doesn't seem relevant.

**Comment by mohanarpit** [comment]
@deepakchethan Can you please also make an entry to `Summary.md` ? Else this section will not show up in docs.appsmith.com

File: function-reference/store-value.md (hunk 1 above)
Comment (deepakchethan) [reply function-reference/store-value.md]: Done

**Comment by deepakchethan** [comment]
@mohanarpit, I have added the reference as mentioned

**Review by hetunandu — APPROVED** [review]

**Pull request merged**